analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -198,6 +205,33 @@ private void writeGmeMarkerFile() throws IOException, ManipulationException {
         }
     }
 
+    private void writeGmeReposMarkerFile() throws IOException {
+        File rootDir = getProject().getRootDir();
+        File gmeReposGradle = new File(rootDir, GME_REPOS);
+
+        Files.copy(getClass().getResourceAsStream('/' + GME_REPOS), gmeReposGradle.toPath(),
+                StandardCopyOption.REPLACE_EXISTING);
+    }
+
+    private void updateAllExtraGradleFilesWithGmeRepos() throws IOException {
+        final File rootDir = getProject().getRootDir();
+        final File gradleScriptsDirectory = rootDir.toPath().resolve("gradle").toFile();
+        if (!gradleScriptsDirectory.exists()) {
+            return;
+        }
+        final Collection<File> extraGradleScripts = FileUtils.listFiles(gradleScriptsDirectory, new SuffixFileFilter(".gradle"),
+                DirectoryFileFilter.DIRECTORY);
+        for (File extraGradleScript : extraGradleScripts) {
+            final List<String> lines = FileUtils.readLines(extraGradleScript, Charset.defaultCharset());
+
```

Comment:
Don't we need to use `String first = org.jboss.gm.common.utils.FileUtils.getFirstLine(lines); ` so we don't end up with duplicate lines?